app/src/main/res/layout/activity_drawing.xml

```diff
@@ -8,17 +8,17 @@
 
     <LinearLayout
         android:layout_width="match_parent"
-        android:layout_height="0dp"
-        android:layout_weight="1"
+        android:layout_height="wrap_content"
+        android:layout_weight=".15"
```

Comment:
If you set the layout_height to wrap_content, the weight is useless. The weight insures a good ratio between the drawing, the tool and the color sections. With this change, if you have a high resolution the tools and colors will be very small

Reply:
There were minor changes in this layout because I first thought to extend the DrawingOnline activity, but then moved everything over to DrawingOffline.
I now replaced this layout with the one from master again.